java/datastore/src/main/java/com/google/datastore/v1/client/RemoteRpc.java

```diff
@@ -49,6 +51,21 @@
   private final HttpRequestInitializer initializer;
   private final String url;
   private final AtomicInteger rpcCount = new AtomicInteger(0);
+  private static boolean enableE2EChecksum;
```

Comment:
This can be final

Reply:
forgot to include a comment saying "not final because unittests want to set/reset this value"